lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -569,50 +573,54 @@ static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
 
   if (!self_type.IsValid() ||
       !self_type.GetTypeSystem()->SupportsLanguage(lldb::eLanguageTypeSwift))
-    return;
+    return Status("Could not add required aliases: self is not valid.");
```

Comment:
```suggestion
    return Status("Unable to add the aliases the expression needs because `self` isn't valid.");
```

If the word "Unable" isn't a valid option, then I suggest the contraction "Couldn't".